hack/meridio-init.sh

```diff
@@ -0,0 +1,212 @@
+	ip link add $dev type vxlan $VXLAN_PARAMS || die "setup_vxlan $VXLAN_PARAMS"
+	ip link set up dev $dev
+	ip addr add 10.30.30.0/30 dev $dev
+}
+##   lb
+##     Setup for Meridio stateless-lb. If $TUNNEL_PARAMS is specified
+##     like TUNNEL_PARAMS=<peer>[:port],<id>[,<ip1>,ip2] Then a tunnel device
+##     is setup with local device $NFE_EXTERNAL_INTERFACE or
+##     ext-vlan.$id by default.  No spaces allowed. Example;
+##
+##       TUNNEL_PARAMS=192.168.1.201:5533,333,10.30.30.1/30,fd00:1::/64
+##       TUNNEL_PARAMS=[fc00:1111::1],100
+##
+cmd_lb() {
+	log "Setup for Meridio stateless-lb"
+	if test "$__ipv6_only" != "yes"; then
+		sysctl -w net.ipv4.conf.all.forwarding=1
+		sysctl -w net.ipv4.fib_multipath_hash_policy=1
+		sysctl -w net.ipv4.conf.all.rp_filter=0
+		sysctl -w net.ipv4.conf.default.rp_filter=0
+	fi
+	if test "$__ipv4_only" != "yes"; then
+		sysctl -w net.ipv6.fib_multipath_hash_policy=1
+		sysctl -w net.ipv6.conf.all.forwarding=1
+	fi
+	test -n "$TUNNEL_PARAMS" || return 0
+
+	# Setup tunnel interface
+	log "Setting up tunnel with params [$TUNNEL_PARAMS]"
+	local peer id ip1 ip2 cmd dev sport1
+
+	# Get/check params
+	peer=$(echo $TUNNEL_PARAMS,,,, | cut -d, -f1)
+	id=$(echo $TUNNEL_PARAMS,,,, | cut -d, -f2)
+	ip1=$(echo $TUNNEL_PARAMS,,,, | cut -d, -f3)
+	ip2=$(echo $TUNNEL_PARAMS,,,, | cut -d, -f4)
+	log "peer=$peer, ip=$id, ip1=$ip1, ip2=$ip2"
+	test -n "$id" || die "No id"
+	split_address $peer
+	if test -z "$port" || -z "$ip1"; then
+		test $id -ge 255 && die "Id must be <255 for generated port/ip"
+		test -n "$port" || port=$((5500 + id))
+		test -n "$ip1" || ip1=10.30.$id.1/30
+	fi
+	dev=$NFE_EXTERNAL_INTERFACE
+	test -n "$dev" || dev=ext-vlan.$id
+
+	local sport1=$((port + 1))
+	cmd="ip link add $dev type vxlan id $id dev eth0 remote $adr dstport $port srcport $port $sport1"
+	log "$cmd"
+	$cmd || die "Setup failed"
+	ip link set up dev $dev || die "Dev up $dev"
+
+	local ip
+	for ip in $ip1 $ip2; do
+		if echo "$ip" | grep -q : ; then
+			ip -6 addr replace $ip dev $dev || die "Add address $ip"
+		else
+			ip addr replace $ip dev $dev || die "Add address $ip"
+		fi
+	done
+
+	return 0
+}
+##   proxy
+##     Setup for Meridio proxy
+cmd_proxy() {
+	log "Setup for Meridio proxy"
+	if test "$__ipv6_only" != "yes"; then
+		sysctl -w net.ipv4.conf.all.forwarding=1
+		sysctl -w net.ipv4.fib_multipath_hash_policy=1
+		sysctl -w net.ipv4.conf.all.rp_filter=0
+		sysctl -w net.ipv4.conf.default.rp_filter=0
+	fi
+	if test "$__ipv4_only" != "yes"; then
+		sysctl -w net.ipv6.conf.all.forwarding=1
+		sysctl -w net.ipv6.conf.all.accept_dad=0
+		sysctl -w net.ipv6.fib_multipath_hash_policy=1
+	fi
+}
+##   tapa
+##     Setup for Meridio TAPA
+cmd_tapa() {
+	log "Setup for Meridio tapa"
+	sysctl -w net.ipv4.conf.all.arp_announce=2
+}
+
+
+##
+if test -n "$1"; then
+	cmd=$1
+	shift
+else
+	cmd=$INIT_FUNCTION
+fi
+if ! test -n "$cmd"; then
+	# Make a guess from the hostname
+	if hostname | grep -qE 'load-balance|stateless-lb'; then
```

Comment:
I don't know what users will use as attractor and conduit name, so I would more consider undefined "cmd" as a miss-configuration.

Reply:
I have removed the hostname guessing. I wasn't happy with it myself.